back-end/src/main/java/com/soen343/muse/repositories/SongRepo.java

```diff
@@ -6,6 +6,5 @@
 
 public interface SongRepo extends MongoRepository<Song, String>
 {
-     Song findBySongName(String title);
      List<Song> findByArtist(String artist);
```

Comment:
I mean we either keep findByTitle or remove both

Reply:
Fixed